docs/content/tutorials/kic/deploy-with-kic.md

```diff
@@ -100,17 +100,21 @@ To configure NGINX Plus Ingress Controller to communicate with mesh workloads ov
     - The `nginx-plus` argument is required since this feature is only available with NGINX Plus. If you do not specify this flag, the Ingress Controller will fail to start.
     - The `spire-agent-address` passes the address of the SPIRE agent `/run/spire/sockets/agent.sock` to the Ingress Controller.
 
-1. Add NGINX Service Mesh annotation
+1. Add NGINX Service Mesh label
 
-    The following annotation must be added to the Ingress Controller's Pod spec:
+    The following label must be added to the Ingress Controller's Pod spec:
 
     ```yaml
-    annotations:
+    labels:
       nsm.nginx.com/enable-ingress: "true"
       ...
     ```
 
-    This annotation prevents NGINX Service Mesh from automatically injecting the sidecar into the Ingress Controller Pod.
+    This label prevents NGINX Service Mesh from automatically injecting the sidecar into the Ingress Controller Pod.
+
+    {{<note>}}In NGINX Service Mesh versions prior to v1.7.0, the `nsm.nginx.com/enable-ingress: "true"` annotation was used instead of a label.
+    Support for this annotation will be removed in a future release.
+    {{</note>}}
```

Comment:
Screenshot of changes from local testing.
![Screen Shot 2022-12-30 at 11 47 22 AM](https://user-images.githubusercontent.com/50845045/210107533-857d0b96-509b-423f-801a-e173b358dd89.png)